Documentation/CRDs/ceph-client-crd.md

```diff
@@ -5,39 +5,139 @@ title: Client CRD
+```
+
+### 3. Extract Ceph cluster credentials from the generated secret
+
+Extract Ceph cluster credentials from the generated secret (note that the subkey will be your original client name):
+
+```console
+kubectl --namespace rook-ceph get secret rook-ceph-client-example -o jsonpath="{.data.example}" | base64 -d
+```
+
+The base64 encoded value that is returned *is* the password for your ceph client.
+
+### 4. Retrieve the mon endpoints
+
+To send writes to the cluster, you must retrieve the mons in use:
+
+```console
+kubectl --namespace rook-ceph get configmap rook-ceph-mon-endpoints -o jsonpath='{.data.data}' | sed 's/.=//g'`
+```
+
+This command *should* produce a line that looks somewhat like this:
+
+```
+10.107.72.122:6789,10.103.244.218:6789,10.99.33.227:6789
+```
+
+### 5. (optional) Generate Ceph configuration files
+
+If you choose to generate files for Ceph to use you will need to generate the following files:
+
+- General configuration file (ex. `ceph.conf`)
+- Keyring file (ex. `ceph.keyring`)
+
+Examples of the files follow:
+
+`ceph.conf`
+```ini
+[global]
+mon_host=10.107.72.122:6789,10.103.244.218:6789,10.99.33.227:6789
+log file = /tmp/ceph-$pid.log
+```
+
+`ceph.keyring`
+```ini
+[client.example]
+  key = < key, decoded from k8s secret>
+  # The caps below are for a rbd workload -- you may need to edit/modify these capabilities for other workloads
+  # see https://docs.ceph.com/en/latest/cephfs/capabilities
+  caps mon = 'allow r'
+  caps osd = 'profile rbd pool=<your pool>, profile rb pool=<another pool>'
+```
+
+### 6. Connect to the Ceph cluster with your given client ID
+
+With the files we've created, you should be able to query the cluster by setting Ceph ENV variables and running `ceph status`:
+
+```
+export CEPH_CONF=/libsqliteceph/ceph.conf;
+export CEPH_KEYRING=/libsqliteceph/ceph.keyring;
+export CEPH_ARGS=--id example;
+ceph status
+```
+
+With this config, the ceph tools (`ceph` CLI, in-program access, etc) can connect to and utilize the Ceph cluster.
+
+## Use Case: SQLite
+
+The Ceph project contains a [SQLite VFS][sqlite-vfs] that interacts with RBD directly, called [`libcephsqlite`][libcephsqlite].
+
+First, on your workload ensure that you have the appropriate packages installed that make `libcephsqlite.so` available:
+
+- [`ceph` on Alpine](https://pkgs.alpinelinux.org/package/edge/community/x86_64/ceph)
+- [`libsqlite3-mod-ceph` on Ubuntu](https://pkgs.alpinelinux.org/package/edge/community/x86_64/ceph)
+- [`libcephsqlite` on Fedora](https://pkgs.org/search/?q=libcephsqlite)
+- [`ceph` on CentOS](https://cbs.centos.org/koji/packageinfo?packageID=534)
+
+Without the appropriate package (or a from-scratch build of SQLite), you will be unable to load `libcephsqlite.so`.
+
+After creating a `CephClient` similar to [`deploy/examples/sqlitevfs-client.yaml`](../../deploy/examples/sqlitevfs-client.yaml) and retrieving it's credentials, you may set the following ENV variables:
```

Comment:
The docs are rendered when published to rook.io, so the full URL is needed like so. It's a pain that we can't test the link before merge, but it will be fine after merge.
```suggestion
After creating a `CephClient` similar to [`deploy/examples/sqlitevfs-client.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/sqlitevfs-client.yaml) and retrieving it's credentials, you may set the following ENV variables:
```

Reply:
- [ ] fix link to `sqlitevfs-client.yaml`